src/github.com/getlantern/go-loggly/loggly.go

```diff
@@ -0,0 +1,328 @@
+package loggly
+
+import . "github.com/visionmedia/go-debug"
+import . "encoding/json"
+import "io/ioutil"
+import "net/http"
+import "strings"
+import "bytes"
+import "time"
+import "sync"
+import "fmt"
+import "os"
+import "io"
+
+const Version = "0.4.3"
+
+const api = "https://logs-01.loggly.com/bulk/{token}"
+
+type Message map[string]interface{}
+
+var debug = Debug("loggly")
+
+var nl = []byte{'\n'}
+
+var (
```

Comment:
Adding the http client we're going to use for proxying request and its associated rw mutex.